api/holiday_acres_api/tests/horse_unit_test.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from holiday_acres_api.models import Horse
+from holiday_acres_api.models import Paddock
+from holiday_acres_api.models import User
+
+
+"""
+Can create and fetch basic horse in DB
+"""
+
+
```

Comment:
For testing relationships, let's make that a whole separate test.

Create another function called something like:
```python
def test_horse_relationships():
    ...
```
And test the relationships there

Reply:
Oh. I misread your instructions and created an entirely separate test. I'll fix that and recommit.